Description: Fix to work with wxWidgets 3.2
Author: Scott Talbert <swt@techie.net>
Last-Update: 2022-09-16
Bug-Debian: https://bugs.debian.org/1019787
Forwarded: no

--- a/src/sjbase/skinml.cpp
+++ b/src/sjbase/skinml.cpp
@@ -882,7 +882,7 @@ bool SjSkinMlTagHandler::HandleTag(const
 			// second, execute stuff between <script> and </script>
 			if( tag.HasEnding() )
 			{
-				scriptContent = m_skinMlParser->GetSource()->Mid(tag.GetBeginPos(), tag.GetEndPos1()-tag.GetBeginPos());
+				scriptContent = wxString(tag.GetBeginIter(), tag.GetEndIter1());
 				scriptContent.Trim(true);
 				scriptContent.Trim(false);
 				if( !scriptContent.IsEmpty() )
