--- XaraLX-0.7r1785.orig/Kernel/ccfile.cpp	2021-03-17 22:31:37.550354604 -0300
+++ XaraLX-0.7r1785/Kernel/ccfile.cpp	2021-03-17 22:32:28.404037106 -0300
@@ -448,6 +448,7 @@
 	// Work out which error description string to use...
 	switch (GZipErrorID)
 	{
+/**
 		case Z_ERRNO:
 			// Bad file number
 			ZipFileErrorID = _R(IDE_ZFILE_ERRORNO);
@@ -472,7 +473,7 @@
 			// Bad file number
 			ZipFileErrorID = _R(IDE_ZFILE_BUFFERERROR);
 			break;
-
+**/
 		default:
 			// Unknown error
 			ZipFileErrorID = _R(IDE_ZFILE_BASEEERROR);
