--- p7zip-17.04-orig/CPP/Windows/Control/Controls.cpp	2021-04-04 00:11:06.000000000 -0300
+++ p7zip-17.04/CPP/Windows/Control/Controls.cpp	2021-08-08 23:25:19.680327032 -0300
@@ -83,7 +83,7 @@
                     exit(-1);
                 }
             }
-            s = str;
+            s = static_cast<const char *>(str);
             return true;
         }
     }
@@ -122,7 +122,7 @@
             void CComboBox::GetText(CSysString &s) {
                 LockGUI lock;
                 wxString str = _choice->GetValue();
-                s = str;
+                s = static_cast<const char *>(str);
             }
 
             int CComboBox::GetCount() const  {
@@ -133,7 +133,7 @@
             void CComboBox::GetLBText(int index, CSysString &s) {
                 LockGUI lock;
                 wxString str = _choice->GetString(index);
-                s = str;
+                s = static_cast<const char *>(str);
             }
 
             void CComboBox::SetCurSel(int index) {
@@ -208,7 +208,7 @@
                 LockGUI lock;
                 str = ((wxTextCtrl *)_window)->GetValue();
             }
-            s = str;
+            s = static_cast<const char *>(str);
             return true;
         }
 
