--- p7zip-17.04-orig/CPP/Windows/Control/Dialog.cpp	2021-04-04 00:11:06.000000000 -0300
+++ p7zip-17.04/CPP/Windows/Control/Dialog.cpp	2021-08-08 23:25:47.704333494 -0300
@@ -306,7 +306,7 @@
 					LockGUI lock;
 	  				str = _window->GetTitle();
 				}
-	  			s = str;
+	  			s = static_cast<const char *>(str);
 	  			return true;
 		       	}
 
@@ -441,7 +441,7 @@
 			       	g_tabCreate[n].title, defaultDir);
 		dirDialog.SetIcon(wxICON(p7zip_32));
 		int ret = dirDialog.ShowModal();
-		if (ret == wxID_OK) g_tabCreate[n].resultPath = dirDialog.GetPath();
+		if (ret == wxID_OK) g_tabCreate[n].resultPath = static_cast<const wchar_t *>(dirDialog.GetPath());
 		return ret;
 	}
 
@@ -462,7 +462,7 @@
 				dir, name, wxT("All Files (*.*)|*.*"), wxFD_SAVE|wxFD_OVERWRITE_PROMPT);
 		fileDialog.SetIcon(wxICON(p7zip_32));
 		int ret = fileDialog.ShowModal();
-		if (ret == wxID_OK) g_tabCreate[n].resultPath = fileDialog.GetPath();
+		if (ret == wxID_OK) g_tabCreate[n].resultPath = static_cast<const wchar_t *>(fileDialog.GetPath());
 		return ret;
 	}
 
