--- a/src/about.py	2014-11-03 23:26:09.000000000 -0200
+++ b/src/about.py	2021-10-24 15:10:53.261157542 -0300
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 
 from gi.repository import Gtk
-import os
 from gettext import gettext as _
 
 Authors = ["Christian Schaller <uraeus@gnome.org>",
@@ -58,6 +57,3 @@
        x.set_license(LGPL)   
        x.connect("response", lambda d, r: d.destroy())
        x.show()
-
-   def close(self,widget):
-        sys.exit(0)
